templates/js/custom-copilot-assistant-assistants-api/README.md.tpl

```diff
@@ -67,26 +67,24 @@ Before running or debugging your bot, please follow these steps to setup your ow
    ```
 {{/useOpenAI}}
 {{#useAzureOpenAI}}
-1. After `npm install` completed, run command `npm run assistant:create -- <your-azure-openai-api-key> <your-azure-openai-deployment-name> <your-azure-openai-endpoint>`
+1. After `npm install` completed, run command `npm run assistant:create -- <your-azure-openai-api-key>`
```

Comment:
Remind users to fill in endpoint, deployment name in creator, in case they do not fill in them when scaffolding.

Reply:
sure, added